File: haushaltsbuch/src/test/java/org/github/haschi/haushaltsbuch/Inventarposition.java
package org.github.haschi.haushaltsbuch;


import cucumber.deps.com.thoughtworks.xstream.annotations.XStreamConverter;
import org.github.haschi.haushaltsbuch.api.Währungsbetrag;
import org.github.haschi.infrastruktur.MoneyConverter;

final class Inventarposition
{
    String gruppe;
    String untergruppe;
    String position;
    @XStreamConverter(MoneyConverter.class)
    Währungsbetrag währungsbetrag;
}

File: haushaltsbuch/src/test/kotlin/org/github/haschi/haushaltsbuch/Inventarposition.kt
package org.github.haschi.haushaltsbuch


import cucumber.deps.com.thoughtworks.xstream.annotations.XStreamConverter
import org.github.haschi.haushaltsbuch.core.values.Währungsbetrag
import org.github.haschi.infrastruktur.MoneyConverter

class Inventarposition {
    var gruppe: String? = null
    var untergruppe: String? = null
    var position: String? = null
    @XStreamConverter(MoneyConverter::class)
    var währungsbetrag: Währungsbetrag? = null
}
